test_system.py

```diff
@@ -0,0 +1,37 @@
+import threading
+import time
+import web_api as web_server
+from explainer import process_files
+from web_app_client import WebAppClient
+
+
+def test_system():
```

Comment:
Separate code to small functions.  Read more: https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be